lib/grape-swagger/endpoint.rb

```diff
@@ -198,7 +198,7 @@ def response_object(route)
         response_model = expose_params_from_model(value[:model]) if value[:model]
 
         if route.request_method == 'DELETE' && !value[:model].nil?
```

Comment:
sorry, i'm still confused about how you'd like to address the problem.  i'm willing to make alterations to this pull request.  do you want to:

a) Assert compatibility via a dependency lock on < 0.19.0, and make the changes specific to that version
b) Revert the change that I mention above and restore compatibility with all versions
c) Something else?

Sorry for being confused...